ui/browser/views/app.jsx

```diff
@@ -17,11 +17,12 @@ import BrowserWindow from './browser.jsx';
 
 export const App = function({ state, dispatch }) {
   return (
-    <BrowserWindow pages={state.pages}
-      pageOrder={state.pageOrder}
-      currentPageIndex={state.currentPageIndex}
-      pageAreaVisible={state.pageAreaVisible}
+    <BrowserWindow pages={state.window.pages}
```

Comment:
On Thu, Apr 14, 2016 at 6:05 PM, Jordan Santell <notifications@github.com>
wrote:

> In ui/browser/views/app.jsx
> <https://github.com/mozilla/tofino/pull/130#discussion_r59815570>:
>
> > @@ -17,11 +17,12 @@ import BrowserWindow from './browser.jsx';
> >
> >  export const App = function({ state, dispatch }) {
> >    return (
> > -    <BrowserWindow pages={state.pages}
> > -      pageOrder={state.pageOrder}
> > -      currentPageIndex={state.currentPageIndex}
> > -      pageAreaVisible={state.pageAreaVisible}
> > +    <BrowserWindow pages={state.window.pages}
>
> window will make this strange when destructuring in a browser context
>
Ah, right.  Feel free to counter, but `basic` feels wrong.